load_collection.json

```diff
@@ -268,13 +303,19 @@
                                     }
                                 }
                             }
-                        }
+                        },
+                        "crs": 3857
```

Comment:
I wouldn't put the `crs` parameter in the example, as it would suggest that this is something that should always be used - users should rather be encouraged to leave the CRS choice to the backend.